### DISCUSSION TITLE
API with comments QTY per page

### DISCUSSION THREAD

**Discussion by rez0n** [umputun/remark42#1709] [Q&A] [answered]
Hi, I would like to show comments qty for each post in posts list.
But it seems there are no any endpoint to get this data excluding this
```
GET /api/v1/find?site=site-id&url=post-url&sort=fld&format=tree|plain - find all comments for given post
```
But it sounds extremely inefficient and could lead to performance issues.
Any ideas or suggestions?

**Comment by umputun** [comment, marked as answer]
ins't `GET {{host}}/api/v1/count?site={{site}}&url={{url}}` what you meant?

![image](https://github.com/umputun/remark42/assets/535880/3f0f4e8d-6c76-4921-8192-42592c41388a)

**Reply by rez0n** [reply]
Oh, seems time to visit oculist. 
Thanks!
